config/.eslintrc.json

```diff
@@ -17,8 +17,9 @@
     "overrides": [
         {
             "files": [
-                "*test.ts",
-                "**/test/**/*.ts"
+                "**/*.test.ts",
+                "**/test/**/*.ts",
+                "**/testing/**/*.ts"
```

Comment:
I see code under `testing/` as regular code that is used for testing. I don't think we should disable these two rules under `testing/`.

Reply:
Okay. will remove it.